.devcontainer/devcontainer.json

```diff
@@ -11,12 +11,13 @@
     "mounts": [
         "source=freqtrade-bashhistory,target=/home/ftuser/commandhistory,type=volume"
     ],
+    "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/freqtrade,type=bind,consistency=cached",
     // Uncomment to connect as a non-root user if you've added one. See https://aka.ms/vscode-remote/containers/non-root.
     "remoteUser": "ftuser",
 
-    "postCreateCommand": "freqtrade create-userdir --userdir user_data/",
+    "postCreateCommand": "ln -s /freqtrade/.env /workspaces/freqtrade/.env && freqtrade create-userdir --userdir user_data/",
```

Comment:
the `.env` linking seems wrong (maybe you can explain your reasoning on this??).

Why should we use a virutalenv within a container which has freqtrade installed already (That's the whole point of using the devcontainer - to have all dependencies contained within it).

Assuming you're on windows - then the virtual environment would even be broken (python venv vs. linux docker container).

Reply:
Sure, I will try. If you mount the source folder and not the folder from the docker image, you lose the installation, which is done during docker build. So in this case the installation is done in /freqtrade dir, which has the .env. But the local sources don't have it and that is correct since you don't want to install the dependencies all over again on your local machine, neither through this setup.

Reply:
> Assuming you're on windows - then the virtual environment would even be broken (python venv vs. linux docker container).

It should not be platform specific, because it is a command in a linux container, should work the same on windows and mac and linux

Reply:
the `.env` directory is coming from the host (is mounted) - so this is clearly wrong (you shouldn't activate or use a virtual environment from within the container).

i think IF we're moving the directory from /freqtrade to /workspace/freqtrade - then a simple `pip install --user -e .`on that directory is the proper approach.

Reply:
The correct command to put here (considering the other changes) seems to be `pip install --user -e .`.
That'll make sure the correct freqtrade version is used (not the one in `/freqtrade`).

Reply:
Yes, it seems it is the way to go. This is a very good remark.